src/routes/committee/setlist/+page.svelte

```diff
@@ -155,19 +210,75 @@
 	</ScrollableList>
 </Grid>
 
+<Modal
+	modalHeading="Add to setlist"
```

Comment:
at this point begint het misschien de moeite waard te worden deze in aparte files te definieren

Reply:
Ja twijfelde ik ook al over

Reply:
Zal de rest verplaatsen in een refactor merge

Reply:
oki